pkg/cmd/server/start/admission.go

```diff
@@ -79,6 +78,7 @@ var (
 		imagepolicy.PluginName,
 		"BuildOverrides",
 		"AlwaysPullImages",
+		"openshift.io/OwnerReference",
```

Comment:
follow-up issue to make the admission plugin check delete permissions and re-enable it